benchmark.py

```diff
@@ -176,6 +177,35 @@ def RunBenchmark(args, logfile):
   else:
     print "Instance deletion is skipped due to --no-create_delete."
 
+  data_store_command = ("python store_data.py --ownip {}"
+                        " --runid {} --envoy_hash {}").format(
+                            args.ownip, args.runid, args.envoy_hash)
+
+  if args.create_db_instance:
+    data_store_command = ("{} --create_instance --db_instance_name {}").format(
+        data_store_command, args.db_instance_name
+    )
+  else:
+    data_store_command = "{} --no-create_instance".format(data_store_command)
+
+  if args.create_db:
+    data_store_command = ("{} --create_db --database {}").format(
+        data_store_command, args.database
+    )
+  else:
+    data_store_command = "{} --no-create_db".format(data_store_command)
+
+  if args.delete_db:
+    data_store_command = ("{} --delete_db").format(
+        data_store_command
+    )
+  else:
+    data_store_command = "{} --no-delete_db".format(data_store_command)
+
+  print data_store_command
+  pexpect.run(data_store_command, timeout=None, logfile=logfile)
+  print "Data stored into database."
```

Comment:
Can you file an issue to convert all these `print` statement to https://docs.python.org/2.7/library/logging.html? We should move away from doing `print` and do properly qualified (in terms of verbosity, warning/error/info/debug etc.) log messages.